src/run.ts

```diff
@@ -55,11 +57,16 @@ async function runLint(lintPath: string): Promise<void> {
     throw new Error(`please, don't change out-format for golangci-lint: it can be broken in a future`)
   }
 
+  const workingDirectory = path.resolve(core.getInput(`working-directory`))
```

Comment:
will it work if user didn't specify `working-directory`? maybe we should check that the input was set?

Reply:
It will work because `getInput` returns `''`. This behaviour is tested in the `actions/core` test suite so I don't expect it to change.

Resolving the empty string relative to the current working directory, returns the current working directory.

I can make this more explicit if you wish though.

Reply:
thank you, I would prefer more explicit logic or unit test on it. I'm not sure that this behavior of `path.resolve` is guaranteed and stable.

Reply:
I'm certain it will. Because:

> If after processing all given path segments an absolute path has not yet been generated, the current working directory is used.

but sure, I'll make this explicit.

Reply:
Done!